x-pack/plugins/infra/public/pages/logs/log_entry_rate/sections/anomalies/expanded_row.tsx

```diff
@@ -4,86 +4,130 @@
+
+  const {
+    getLogEntryRateExamples,
+    hasFailedLoadingLogEntryRateExamples,
+    isLoadingLogEntryRateExamples,
+    logEntryRateExamples,
+  } = useLogEntryRateExamples({
+    dataset: anomaly.partitionId,
+    endTime: anomaly.startTime + bucketSpan / 2,
+    exampleCount: EXAMPLE_COUNT,
+    sourceId,
+    startTime: anomaly.startTime - bucketSpan / 2,
+  });
+
+  useMount(() => {
+    getLogEntryRateExamples();
+  });
+
   return (
-    <EuiFlexGroup>
-      <EuiFlexItem grow={8}>
-        <AnomaliesChart
-          chartId={`${partitionId}-anomalies`}
-          timeRange={timeRange}
-          setTimeRange={setTimeRange}
-          series={logEntryRateSeries}
-          annotations={anomalyAnnotations}
-        />
-      </EuiFlexItem>
-      <EuiFlexItem>
-        <EuiSpacer size="m" />
-        <EuiStat
-          title={numeral(totalNumberOfLogEntries).format('0.00a')}
-          titleSize="m"
-          description={i18n.translate(
-            'xpack.infra.logs.analysis.anomaliesExpandedRowNumberOfLogEntriesDescription',
-            {
-              defaultMessage: 'Number of log entries',
-            }
-          )}
-          reverse
-        />
-        <EuiSpacer size="m" />
-        <EuiFlexGroup>
-          <EuiFlexItem grow={false}>
-            <AnalyzeInMlButton jobId={jobId} timeRange={timeRange} partition={partitionId} />
-          </EuiFlexItem>
-        </EuiFlexGroup>
-      </EuiFlexItem>
-    </EuiFlexGroup>
+    <>
+      <ExpandedContentWrapper direction="column">
+        <EuiFlexItem>
+          <EuiTitle size="s">
+            <h3>{examplesTitle}</h3>
+          </EuiTitle>
+          <LogEntryExampleMessages
+            isLoading={isLoadingLogEntryRateExamples}
+            hasFailedLoading={hasFailedLoadingLogEntryRateExamples}
+            hasResults={logEntryRateExamples.length > 0}
+            exampleCount={EXAMPLE_COUNT}
+            onReload={getLogEntryRateExamples}
+          >
+            {logEntryRateExamples.length > 0 ? (
+              <>
+                <LogEntryRateExampleMessageHeaders dateTime={logEntryRateExamples[0].timestamp} />
+                {logEntryRateExamples.map((example, exampleIndex) => (
+                  <LogEntryRateExampleMessage
+                    key={exampleIndex}
+                    id={example.id}
+                    dataset={example.dataset}
+                    message={example.message}
+                    timestamp={example.timestamp}
+                    tiebreaker={example.tiebreaker}
+                    timeRange={timeRange}
+                    jobId={jobId}
+                  />
+                ))}
+              </>
+            ) : null}
+          </LogEntryExampleMessages>
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiFlexGroup>
+            <EuiFlexItem grow={false}>
+              <EuiStat
+                titleSize="s"
+                title={`${numeral(anomaly.typicalLogEntryRate).format('0.00a')} ${i18n.translate(
```

Comment:
:thought_balloon: I wondered whether the order of "`<count>` `<itemtype>`" is consistent across locales. So I tried formatting it in the abstract via the ICU syntax `{typicalCount, number, ::compact-short .00}`, which should have worked. But I wasted an hour finding out that it doesn't work for us because the versions of our translation libs seem to be too old. :disappointed: 

So nothing we can do about it for now, it seems.

Reply:
😞 Shame, thanks for pointing out those things though.